includes/REST_API/Stories_Controller.php

```diff
@@ -320,11 +320,42 @@ public function get_items( $request ) {
 			$posts_query->query( $query_args );
 			$statuses_count[ $key ] = absint( $posts_query->found_posts );
 		}
+
+		if ( $request['_web_stories_envelope'] ) {
+			$current_data = $response->get_data();
+			$headers      = $response->get_headers();
+			$data         = [
+				'totals'  => $statuses_count,
```

Comment:
Can't this just be accessed via `headers`?

Reply:
The dashboard app checks `response.headers` anyway, so this seems superfluous.

Reply:
Fixed https://github.com/google/web-stories-wp/pull/2933/commits/1005ffd39f2781976e5737c390d4b212ca099c17